### PR TITLE
Update sns filter

### PR DESCRIPTION
**Pull request by givanovexpe**

### CODE REVIEW

**Comment by givanovexpe** [comment]
Sorry guys, I don't know why but somehow the PR disappeared. I've amended the code to all the latest comments.

File: privileges-grantor/README.md [hunk 1]
```diff
@@ -11,8 +11,7 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | instance\_name | Privileges Grantor instance name to identify resources in multi-instance deployments. | string | `""` | no |
 | lambda\_timeout | The time after which the lambda execution stops. | string | `"200"` | no |
 | memory | The amount of memory (in MiB) to be used by Lambda | string | `"512"` | no |
-| metastore\_events\_filter | List of metastore event types to be added to SNS filter. Supported format: `[ "CREATE_TABLE","ALTER_TABLE" ]` | list | `[ "CREATE_TABLE", "ALTER_TABLE" ]` | no |
-| database\_filter | List of database names to be added to SNS filter. Supported format: `[ "DB_NAME_1", "DB_NAME_2" ]` | list | n/a | yes |
+| metastore\_events\_sns\_filter | SNS filter of message attributes to the added to the sns sns topic subscription. Supported format: SNS filter format. Refer to https://docs.aws.amazon.com/sns/latest/dg/sns-message-filtering.html for more information on how to construct a filter. | string | `{ "eventType": [ "CREATE_TABLE", "ALTER_TABLE" ] }` | no |
```
Comment (massdosage) [inline privileges-grantor/README.md]:
```suggestion
| metastore\_events\_sns\_filter | SNS filter of message attributes to the added to the SNS topic subscription. Supported format: SNS filter format. Refer to https://docs.aws.amazon.com/sns/latest/dg/sns-message-filtering.html for more information on how to construct a filter. | string | `{ "eventType": [ "CREATE_TABLE", "ALTER_TABLE" ] }` | no |
```

**Review by massdosage — CHANGES_REQUESTED** [review]
Just one small typo left, rest looks good.

**Review by massdosage — APPROVED** [review]

**Review by abhimanyugupta07 — APPROVED** [review]

File: privileges-grantor/README.md [hunk 2]
```diff
@@ -49,38 +60,53 @@ The jars can be downloaded from the link provided above and uploaded to S3 via t
 variable "pg_lambda_version" {
   description = "Version of the Privilege Grantor Lambda."
   type        = "string"
-  default     = "4.2.0"
+  default     = "6.1.0"
```
Comment (abhimanyugupta07) [inline privileges-grantor/README.md:63]: The apiary-extensions version is 6.0.1.

**Pull request merged**